Bluepill-runner/Bluepill-runner/BPRunner.m

```diff
@@ -123,25 +123,17 @@ - (int)run {
     signal(SIGINT, onInterrupt);
     
     NSUInteger numSims = [self.config.numSims intValue];
-    NSUInteger origNumSims = numSims;
     [BPUtils printInfo:INFO withString:@"This is Bluepill %s", BP_VERSION];
     NSError *error;
-    NSMutableArray *bundles = [BPPacker packTests:self.app.testBundles withNoSplitList:self.config.noSplit intoBundles:numSims andError:&error];
-    while (!bundles && numSims > 1) {
```

Comment:
I changed the `packTests` method to just use the maximum number of simulators possible in that case instead of throwing an error and retrying until we finally hit the right number (https://github.com/linkedin/bluepill/pull/39/files#diff-34fb18c19179ad01a103bf483edf2135R48). It will still print the warning if the number of simulators doesn't match up on line 133 of this file (but I should probably change `!=` to `<`).